workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RenderWorkflow.kt

```diff
@@ -186,11 +187,42 @@ public fun <PropsT, OutputT, RenderingT> renderWorkflowIn(
       // we don't surprise anyone with an unexpected rendering pass. Show's over, go home.
       if (!isActive) return@launch
 
+      var drainingActionResult = actionResult
+      if (runtimeConfig.contains(DRAIN_EXCLUSIVE_ACTIONS)) {
+        drain@ while (isActive &&
+          drainingActionResult is ActionApplied<*> &&
+          drainingActionResult.output == null
+        ) {
+          // We start by yielding, because if we are on an Unconfined dispatcher, we want to give
+          // other signals (like Workers listening to the same result) a chance to get dispatched
+          // and queue their actions.
+          yield()
```

Comment:
Passes the sniff test: This is a "tight" loop that doesn't necessarily suspend (I think `sendOutput` is the only suspending call?), so yielding in the middle seems reasonable even if we weren't using an unconfined dispatcher.